Source/SwiftLintFramework/Extensions/SyntaxClassification+isComment.swift

```diff
@@ -0,0 +1,15 @@
+import IDEUtils
+
+extension SyntaxClassification {
+    // True if it is any kind of comment.
+    var isComment: Bool {
+        switch self {
+        case .lineComment, .docLineComment, .blockComment, .docBlockComment:
+            return true
+        case .none, .keyword, .identifier, .typeIdentifier, .operatorIdentifier, .dollarIdentifier, .integerLiteral,
+             .floatingLiteral, .stringLiteral, .stringInterpolationAnchor, .poundDirectiveKeyword, .buildConfigId,
+             .attribute, .objectLiteral, .editorPlaceholder:
```

Comment:
Why the full set of cases here? To make sure the compiler warns if there are new kinds?

Reply:
Exactly 💯